params/governance_paramset_test.go

```diff
@@ -365,6 +366,7 @@ func TestGovParamSet_ChainConfig(t *testing.T) {
 					Reward: &RewardConfig{
 						MintingAmount:          new(big.Int).SetUint64(9.6e18),
 						Ratio:                  "34/54/12",
+						Kip82Ratio:             "20/80",
```

Comment:
It may be changed by default, also.

Reply:
It's reverted back to 20/80 in 0b3e45e

Reply:
You can check https://github.com/klaytn/klaytn/pull/1631#discussion_r1024772248